### DISCUSSION TITLE
Ne söylerdiniz sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#69] [Zeka Sorusu]
Diyelim ki kıyamet yaklaştı; ahlak bozuldu ve size şöyle dedim: “Yalan söylersen iki ödülden biri senin olacak; doğru söylersen ödül yok”. Ödül 1’i alabilmek için ne söylerdiniz?

**Comment by HabilCcabbar** [comment]
**"Ödül 1'i istemiyorum."**
Ödül 1'i almaya çalışıyorsak ödül 1'i istiyoruz. Ve eğer böyle yalan söylersek diğer kişi bizim ödül 1'i istediğimizi anlar ve bu şekilde bize ödül 1'i verir.

**Comment by slos8888** [comment]
**Ödül 2'yi istiyorum veya ödül 1'i istemiyorum diyebiliriz böylece yalan söylemiş oluruz ve ödülü alırız**

**Comment by Furkanyildirm** [comment]
yalan söylemediğimde ödül 1 benim olacak dediğimde yalan söylediğim ve ödül 1'i istediğim anlaşılacak
